components/contract-components/releaser/releaser-header.tsx

```diff
@@ -34,12 +36,23 @@ export const ReleaserHeader: React.FC<ReleaserHeaderProps> = ({
           {isProfilePage ? "Author" : "Released by"}
         </Heading>
         <Flex gap={4} alignItems="center">
-          <ChakraNextImage
-            alt=""
-            boxSize={12}
-            mt={1}
-            src={require("public/assets/others/hexagon.png")}
-          />
+          {releaserProfile.data?.avatar ? (
+            <ChakraMediaRenderer
+              alt=""
+              boxSize={12}
+              mt={1}
+              src={releaserProfile.data?.avatar}
+            />
+          ) : (
+            <ChakraNextImage
+              alt=""
+              boxSize={12}
+              width="100%"
+              height="100%"
+              mt={1}
+              src={`https://source.boringavatars.com/marble/120/${wallet}?colors=264653,2a9d8f,e9c46a,f4a261,e76f51`}
```

Comment:
I think we can just use a regular old image for this, instead. Meaning we don't have to optimize these images.